### ISSUE TITLE
dbus-api should have a signal for new_record

### ISSUE THREAD

**Issue by MaZderMind** [timvideos/gst-switch#196]
The signal shoud be raised whenever a client requests a new recording to be startet. It can be used by other Clients to update their UI or to start processing the now finished recording.

**Comment by mithro** [comment]
Agreed. Would be good if the signal included the new file name too.
